packages/plugin-apex/messages/tail.json

```diff
@@ -0,0 +1,11 @@
+{
+    "commandDescription": "follows active log",
+    "longDescription": "Activates debug logging and displays logs in the terminal. You can also pipe the logs to a file.",
+    "jsonDescription": "Format output as JSON.",
+    "logLevelDescription": "logging level for this command invocation",
```

Comment:
Should 'logging level...` be capitalized? (eg `Logging level...`)

Reply:
Sure, I think that's a good cleanup opportunity here!